src/paths.cpp

```
@@ -165,4 +165,35 @@ void write_manifest(const fs::path &dir,
 }
 
+fs::path find_native_library(const std::string &exe_path,
+                              const std::string &lib_name) {
+#ifdef __APPLE__
+    std::string filename = "libry_" + lib_name + ".dylib";
+#else
+    std::string filename = "libry_" + lib_name + ".so";
+#endif
+
+    std::error_code ec;
+    fs::path exe_dir = fs::path(exe_path).parent_path();
```

Clang (14.0.6)

[warning] variable 'exe_dir' is not initialized

(cppcoreguidelines-init-variables)